frontend/packages/console-shared/locales/en/console-shared.json

```diff
@@ -83,8 +83,8 @@
   "in": "in",
   "out": "out",
   "View {{title}} metrics in query browser": "View {{title}} metrics in query browser",
-  "{{humanAvailable}} available of {{humanLimit}} total limit": "{{humanAvailable}} available of {{humanLimit}} total limit",
-  "{{humanAvailable}} available of {{humanMax}}": "{{humanAvailable}} available of {{humanMax}}",
+  "{{humanAvailable}} available {{humanLimit}} total limit": "{{humanAvailable}} available {{humanLimit}} total limit",
+  "{{humanAvailable}} available {{humanMax}}": "{{humanAvailable}} available {{humanMax}}",
```

Comment:
Not sure if the change also applies here, since Im not a native speaker.

Reply:
I believe it does.

Reply:
I believe it does.